packages/admin/src/form/SubmitResult.tsx

```diff
@@ -0,0 +1,17 @@
+interface SubmitErrorProps<TSubmitError = unknown> {
+    message?: string;
+    submitError?: TSubmitError;
+}
+
+export class SubmitError<TSubmitError = unknown> extends Error {
+    submitError?: TSubmitError;
+
+    constructor(props?: SubmitErrorProps<TSubmitError>) {
```

Comment:
mach bitte message als 1. argument und submitError als 2. 

das ist bei Errors meistens so dass message (wenns eine gibt) nicht in einem obj verschachtelt ist